sql/driver/driver_test.go

```diff
@@ -31,7 +32,7 @@ var isError = testutils.IsError
 
 func setup(t *testing.T) (*server.TestServer, *sql.DB) {
 	s := server.StartTestServer(nil)
-	db, err := sql.Open("cockroach", "https://root@"+s.ServingAddr()+"?certs=test_certs")
+	db, err := sql.Open("cockroach", "https://"+security.RootUser+"@"+s.ServingAddr()+"?certs=test_certs")
```

Comment:
Do we ever change the value of `security.RootUser`? If not, all of the uses of `security.RootUser` just obfuscate its value for me and add unnecessary/undesirable dependencies.